tobac/utils/internal.py

```diff
@@ -674,3 +680,46 @@ def find_hdim_axes_3D_iris(field_in, vertical_coord=None, vertical_axis=None):
         all_axes[np.logical_not(np.isin(all_axes, [time_axis, vertical_coord_axis]))]
     )
     return output_vals
+
+
+@irispandas_to_xarray
+def detect_latlon_coord_name(in_dataset, latitude_name="auto", longitude_name="auto"):
```

Comment:
**Suggestion:** Would `None` be a better choice than `auto` for the default value and to automatically detect the coord name?

Reply:
I think this is generally right. Changing this to be consistent throughout has been/will be a headache. I think we should do this correctly though, so will work on this.

Reply:
Ok, I switched "auto" to `None` throughout for all uses.